version

```diff
@@ -0,0 +1 @@
+1.0
```

Comment:
It would be beneficial to prefix this version so we can independently tag this and any future versions within the repo.

E.g.:
```
17-1.0
```
or similar.

Please note this will require pipeline updates to support the use of the prefix